confAllChambers.py

```diff
@@ -52,6 +58,8 @@ def launchArgs(shelf,link,slot,run,vt1,vt1bump,config,cName,ztrim):
 
     parser.add_option("--config", action="store_true", dest="config",
                       help="Set Configuration from simple txt files", metavar="config")
+    parser.add_option("--dictConfig", action="store_true", dest="dictConfig", default=False,
+                      help="Configure VFATs to custom chamber_default values", metavar="dictConfig")
```

Comment:
To avoid option bloat here and argument bloat in launchArgs remove entirely the `--dictConfig` option _and_ argument.  

Instead a more elegant solution the code (probably inside launchArgs) should check if the link in question is present in chamber_defaults, if it is then the settings provided in chamber_defaults should be used.  If not the settings provided in vfat_user_functions.py L15:

https://github.com/cms-gem-daq-project/cmsgemos/blob/07d68dc291aac5f51f9e5d12fb9e3bd38c16cac7/gempython/tools/vfat_user_functions_uhal.py#L15

Should be used instead.

For e like:

```
if link not in chamber_defaults.keys():
    Defaults from vfat_user_functions.py line 15 applied
    pass
else:
    Settings stored in chamber_defaults applied
    pass
```